si-id-epu50/si_id_epu50/epu.py

```diff
@@ -1,63 +1,63 @@
+import threading
+import socket
 import logging
-logger = logging.getLogger(__name__)
-import logging.handlers as handlers
-import threading, socket
-from datetime import datetime
+import logging.handlers
+import time
 
 from . import constants as _cte
 from .utils import *
-from .ecodrive import EcoDrive
+from .ecodrive import *
```

Comment:
is it possible to import only referenced symbols? we find it hard to keep track of symbols with wildcard imports.

Reply:
Changed to `from .ecodrive import EcoDrive`